augur/api/config.py

```diff
@@ -37,6 +37,8 @@
 
 from augur.application.logs import ROOT_AUGUR_DIRECTORY
 
+loggy_var=ROOT_AUGUR_DIRECTORY+'/logs'
```

Comment:
@IsaacMilarky I'm assuming this was your change. I think we may need a `/` at the end of the logs dir. I'd at least like to check and make sure before this is merged

Reply:
Confirmed. Trailing slash is stored as of today.

Reply:
@IsaacMilarky Actually this is on the old, `config.py` file. I'm not sure why this was changed. I've simply kept this file around for reference.